admin/class-admin.php

```diff
@@ -14,6 +14,11 @@ class WPSEO_Admin {
 	private $options;
 
 	/**
+	 * @var Yoast_Dashboard_Widget
+	 */
+	public $dashboard_widget;
```

Comment:
Because that way somebody can overwrite/remove the filter if they want to. For example if they want to remove the dashboard widget because it isn't suitable for their clients.